tests/IntegrationTest.php

```diff
@@ -75,6 +103,9 @@ public function registerContainerConfiguration(LoaderInterface $loader)
 
             $container->loadFromExtension('webpack_encore', [
                 'output_path' => __DIR__.'/fixtures/build',
+                'builds' => [
+                    'other' =>  __DIR__.'/fixtures/other-build'
```

Comment:
Instead of `other` - how about `different_build`. The problem is that there is an `other_entry` in the main build, so it's confusing.